source/release-notes.txt

```diff
@@ -10,6 +10,42 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.3.4 Changelog
+---------------
+
+*Released December 11, 2023*
+
+Improvements:
+
+- Improved SQL query conversion quality for the Query Converter. 
+  For details on how to join the Query Converter early access program, 
+  see :ref:`rm-query-converter`.
+- Improved application performance through expanded use of available 
+  memory in suitable server environments.
```

Comment:
Was wondering if we could elaborate a bit more on what is improved here?

Aka is it application stability, sync job performance, sync job resilency or something else? This seems pretty generic.